<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NEMainpage</class>
 <widget class="QWidget" name="NEMainpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pingAllButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>230</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ping All</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="broadcastButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Broadcast</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="msgInput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>291</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="PaintWidget" name="paintWidget" native="true">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>319</y>
     <width>571</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="netLogWidget">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>0</y>
     <width>291</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PaintWidget</class>
   <extends>QWidget</extends>
   <header>gui/paintwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
